### ISSUE TITLE
sdp plugins/network lib? Exception on stopping/restarting shng due to "socket not connected"

### ISSUE THREAD

**Issue by onkelandy** [Morg42/plugins#6]
I think this is more a network.py issue maybe as I get the same error in the knx plugin once in a while..?

```
Sep 10 15:41:11 ERROR    lib.plugin        Plugin 'sdp_pioneer' exception in stop() method: [Errno 107] Der Socket ist nicht verbunden
Traceback (most recent call last):
  File "/usr/local/smarthome/lib/plugin.py", line 685, in stop
    self.plugin.stop()
  File "/usr/local/smarthome/lib/model/smartdeviceplugin.py", line 278, in stop
    self._connection.close()
  File "/usr/local/smarthome/lib/model/sdp/connection.py", line 111, in close
    self._close()
  File "/usr/local/smarthome/lib/model/sdp/connection.py", line 450, in _close
    self._tcp.close()
  File "/usr/local/smarthome/lib/network.py", line 925, in close
    self._socket.shutdown(socket.SHUT_RD)
OSError: [Errno 107] Der Socket ist nicht verbunden
```

**Comment by onkelandy** [comment]
Just got that error again. A decent info/debug message instead of the exception would be nice here I think

**Comment by Morg42** [comment]
fixed by smarthomeNG/smarthome#523
